core/templates/services/UpgradedServices.ts

```diff
@@ -81,6 +81,8 @@ import { CamelCaseToHyphensPipe } from
   'filters/string-utility-filters/camel-case-to-hyphens.pipe';
 import { ChangeObjectFactory } from
   'domain/editor/undo_redo/ChangeObjectFactory';
+import { CkEditorCopyContentService } from
```

Comment:
Could you please add this in `oppia-angular-root-component` as well? 

It is described under point 13 here: https://github.com/oppia/oppia/wiki/Angular-Migration#implementation-details-to-migrate-services

Reply:
Done